src/rules/pls_no_land.rs

```diff
@@ -25,25 +26,31 @@ pub fn pls_no_land(paths: &HashSet<PathBuf>) -> anyhow::Result<Vec<diagnostic::D
     Ok(ret)
 }
 
-type LinesView = Vec<String>;
+fn pls_no_land_impl(path: &PathBuf) -> anyhow::Result<Vec<diagnostic::Diagnostic>> {
```

Comment:
I think it's better to accept the reader. The file may not necessary be on file system.

Reply:
This was the signature before, and I believe we get a list of files to lint by the Trunk CLI—is there something else you were thinking of?

Reply:
Oh, sorry I miss understood. I see that you've combined two functions together here. Nvm then